src/main/views/subscription-urn-search-results.njk

```diff
@@ -1,13 +1,14 @@
 {% extends "template.njk" %}
 {% from "govuk/components/table/macro.njk" import govukTable %}
-{% from "./macros/common-components.njk" import submitButton %}
 {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "./macros/common-components.njk" import submitButton %}
 
 {% block pageTitle %}
   {{ title }}
 {% endblock %}
 
 {% block beforeContent %}
+  {{ super() }}
   {{ govukBackLink({
```

Comment:
can we use back from the common components here?

Reply:
refactored